target_chains/solana/sdk/js/pyth_solana_receiver/src/PythSolanaReceiver.ts

```diff
@@ -61,19 +61,26 @@ export type PythTransactionBuilderConfig = {
 
 /**
  * A builder class to build transactions that:
- * - Post price updates (fully or partially verified)
+ * - Post price updates (fully or partially verified) or update price feed accounts
  * - Consume price updates in a consumer program
  * - (Optionally) Close price update and encoded vaa accounts to recover the rent (`closeUpdateAccounts` in `PythTransactionBuilderConfig`)
  *
+ * `addPostPriceUpdates` vs `addUpdatePriceFeed`:
+ * - `addPostPriceUpdates` is used to post price updates to ephemeral accounts. Use this to post a price update from the present or from the past for your program to consume.
+ * - `addUpdatePriceFeed` is used to post price updates to price feed accounts, these are fixed accounts for each feed id that can only be updated with a more recent price update than the one they contain. Their addresses can be found using `getPriceFeedAccountAddress`. Use this to update a shared price feed account with a recent price
+ *
```

Comment:
I think you want to move the more abstract discussion to this comment and then have these lower-level comments on the methods.